requirements/install.txt

```diff
@@ -4,11 +4,11 @@ astropy >= 4.3.1
 cached-property >= 1.5.2
 ipywidgets >= 7.6.5
 matplotlib >= 3.3.0
-numpy >= 1.18.1
+numpy >= 1.19.0
```

Comment:
FYI, `numpy` introduced typing in `v1.20`, [`numpy.typing`](https://numpy.org/devdocs/reference/typing.html).  I'm just saying...

Reply:
WHOA!  That's awesome!  We'd need to keep supporting NumPy 1.19 for the moment to be consistent with [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html), but I'd support requiring 1.20+ starting just after our 0.8.0 release.